metricshub-wbem-extension/src/main/java/org/sentrysoftware/metricshub/extension/wbem/WbemExtension.java

```diff
@@ -181,11 +157,7 @@ public void checkProtocol(TelemetryManager telemetryManager) {
 				);
 			}
 		}
-
-		// Generate a metric from the WBEM result
-		// CHECKSTYLE:OFF
-		metricFactory.collectNumberMetric(hostMonitor, WBEM_UP_METRIC, wbemResult != null ? UP : DOWN, strategyTime);
-		// CHECKSTYLE:ON
+		return false;

```

Comment:
`return wbemResult != null;`

Reply:
we tested before : if wbemResult ! = null
 wbemResult = wbemRequestExecutor.executeWbem( hostname, wbemConfiguration, WBEM_TEST_QUERY, wbemNamespace, telemetryManager ); 
 if (wbemResult != null) { return true }